dotcom-rendering/cypress/plugins/index.js

```diff
@@ -7,16 +7,35 @@
 // You can read more here:
 // https://on.cypress.io/plugins-guide
 // ***********************************************************
+import path from 'path';
 
 const webpackPreprocessor = require('cypress-webpack-preprocessor-v5');
 
 module.exports = (on, config) => {
 	config.env = { ...config.env, ...process.env };
 
 	const webpackConfig = webpackPreprocessor.defaultOptions;
-	webpackConfig.webpackOptions.module.rules[0].exclude =
+	// Adding this here so that we can use a module from source code in the cypress tests
```

Comment:
Needed to update this config so that I was able to access a method from source code in the cypress test

Reply:
So as I understand, this is essentially building DCR inside the cypress app? Or was this happening already and we're just altering it to get it to work?

Reply:
I wonder if it's worthwhile being quite specific about the files we want to import? Though I will admit I'm not massively knowledgeable on cypress best practices

Reply:
Yeah, I believe it is making modules available for importing: https://webpack.js.org/configuration/resolve/ 
Since the only use-case so far is overriding the switches, I think it makes sense to scope it.

Reply:
Found an approach which doesn't require webpack config changes